### PR TITLE
TW-1132 partnership with Gitlab to detect Postman keys

### PR DESCRIPTION
**Pull request by claireriley**

### CODE REVIEW

**Review by Suhas-Gaikwad — APPROVED** [review]
LGTM

File: src/pages/docs/administration/token-scanner.md [hunk 1]
```diff
@@ -86,3 +87,11 @@ To add custom alerts, do the following:
 Postman also works with GitHub to ensure that your Postman API keys are secure. If you commit a valid Postman API key to a public GitHub repository, Postman notifies you by email and in-app notification. You can also set up Postman's [Slack integration](/docs/integrations/available-integrations/slack/) to alert you in Slack if this occurs.
 
 It's recommended you delete the exposed API key in your [API keys dashboard](https://go.postman.co/settings/me/api-keys). You can then [generate a new API key](/docs/developer/postman-api/authentication/#generate-a-postman-api-key) to continue working with the Postman API.
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/token-scanner.md:89]: Suggest formatting this sentence as a tip/note. Consider deleting "It's recommended you" (in other words, start the sentence with the action, "Delete the exposed API key...").

File: src/pages/docs/administration/token-scanner.md [hunk 2]
```diff
@@ -86,3 +87,11 @@ To add custom alerts, do the following:
...
+
+## Protecting Postman API keys in GitLab
+
+> This feature is available on GitLab Ultimate plans.
+
+Postman works with GitLab to protect your Postman API keys in GitLab public repositories. If you accidentally commit a valid Postman API key to a public GitLab repository, Postman notifies you by email and in-app notification.
+
+It's recommended you delete the exposed API key in your [API keys dashboard](https://go.postman.co/settings/me/api-keys) immediately. You can then [generate a new API key](/docs/developer/postman-api/authentication/#generate-a-postman-api-key) to continue working with the Postman API.
```
Comment (nerminamiller-postman) [inline src/pages/docs/administration/token-scanner.md:97]: Consider deleting the previous instance of this sentence.

**Review by nerminamiller-postman — COMMENTED** [review]
Left you a bit of feedback, @claireriley. Looks great!

File: src/pages/docs/administration/token-scanner.md (hunk 1 above)
Comment (claireriley) [reply src/pages/docs/administration/token-scanner.md:89]: We use the "It's recommended you" wording elsewhere so going to keep this for now. In the future we've talked about maybe having different types of notes with colors/icons, so I think until it stands out more I'd prefer to keep this info outside of a note so it's not potentially disregarded

**Pull request merged**